apps/demo/src/length-validator.tsx

```diff
@@ -0,0 +1,45 @@
+import { useValidator } from "@reactway/forms/src";
+import { useCallback } from "react";
+import { ValidationResult } from "@reactway/forms-core/src";
+
+export interface LengthValidatorMessages {
+    tooShort: string;
+    tooLong: string;
+}
+
+export interface LengthValidatorProps {
+    min: number;
+    max?: number;
+    messages: LengthValidatorMessages;
+}
+
+export const LengthValidator: React.FC<LengthValidatorProps> = (props): null => {

```

Comment:
```
export const LengthValidator = (props: LengthValidatorProps): null => {
```

Reply:
So it's defaulting the props inside the component then 😄
https://twitter.com/dan_abramov/status/1133878326358171650